tests/e2e/pageobjects/ide/Ide.ts

```diff
@@ -219,31 +242,19 @@ export class Ide {
     }
 
     async closeAllNotifications() {
-        const notificationLocator: By = By.css('.theia-Notification');
-
+        const statusBarNotificationsLocator: By = By.xpath(`//div[@id="theia-statusBar"]/div[@class="area right"]//div[contains(@title, 'Notification')]`);
+        const notificationCenterActionCloseAllButtonLocator: By = By.xpath(`//div[contains(@class, 'theia-notifications-container theia-notification-center open')]//ul[@class="theia-notification-actions"]/li[@class="clear-all"]`);
+        const notificationCenterClosedLocator: By = By.xpath(`//div[contains(@class, 'theia-notifications-container theia-notification-center closed')]`);
         Logger.debug('Ide.closeAllNotifications');
 
-        if (! await this.driverHelper.isVisible(notificationLocator)) {
-            return;
-        }
-
-        const notifications: WebElement[] = await this.driverHelper.waitAllPresence(notificationLocator);
-        const notificationsCapacity: number = notifications.length;
+        Logger.debug(`Ide.closeAllNotifications driverHelper click Notification Center icon.`);
+        await this.driverHelper.waitAndClick(statusBarNotificationsLocator, 5_000);
 
-        for (let i = 1; i <= notificationsCapacity; i++) {
-            const notificationLocator: By = By.xpath('//div[@class=\'theia-Notification\']//button[text()=\'Close\']');
+        Logger.debug(`Ide.closeAllNotifications driverHelper Notifications Center click closeAllNotifications.`);
+        await this.driverHelper.waitAndClick(notificationCenterActionCloseAllButtonLocator, 5_000);
 
-            try {
-                await this.driverHelper.waitAndClick(notificationLocator);
-            } catch (err) {
-                if (err instanceof error.TimeoutError) {
-                    console.log(`The '${notificationLocator}' element is not visible and can't be clicked`);
-                    continue;
-                }
-
-                throw err;
-            }
-        }
+        Logger.debug(`Ide.closeAllNotifications driverHelper waitng for Notification Center to close.`);
+        await this.driverHelper.waitPresence(notificationCenterClosedLocator, 5_000);
```

Comment:
It would be great to add an additional step and explicitly check that all notifications are closed. Because for now this method checks the behavior of notification center widget. But possible case that after clearing notification center, notifications may still be present.

Reply:
Done.